lib/fog/test_helpers/minitest/assertions.rb

```diff
@@ -0,0 +1,35 @@
+require "fog/schema/data_validator"
+
+# Format related hackery
+# allows both true.is_a?(Fog::Boolean) and false.is_a?(Fog::Boolean)
+# allows both nil.is_a?(Fog::Nullable::String) and ''.is_a?(Fog::Nullable::String)
+module Fog
```

Comment:
do you think you could extract L6-L25 to another file and require it here? The same for https://github.com/fog/fog-core/blob/master/lib/fog/test_helpers/formats_helper.rb Otherwise it's a lot of duplicated code :-)

Something like TypesHelper, or whatever naming you like. Even the FormatsHelper is good, but without the shindo code.